src/Pages/firebase.js

```diff
@@ -0,0 +1,64 @@
+import { db } from '../lib/firebase';
+import { collection, getDocs, addDoc, doc } from 'firebase/firestore';
+import { useEffect, useState } from 'react';
+
+export default function Firebase() {
+  // const [item, setItem] = useState('');
+  const [docs, setDocs] = useState([]);
+
+  useEffect(() => {
+    const getGrocery = async () => {
+      try {
+        const querySnapshot = await getDocs(collection(db, 'groceries'));
+        const snapshotDocs = [];
+        //loop through snapshot & push to array
+        querySnapshot.forEach((doc) => snapshotDocs.push(doc.data()));
+        //set array into docs
+        setDocs(snapshotDocs);
+      } catch (e) {
+        console.log(e.message);
```

Comment:
Nice job with the `try/catch` block! Even if we're not doing much with the error, it's great to see that we're thinking about it. :-)

If you'd like to up the console game, you could use `console.error` here so that it looks like an error in the console.